### PR TITLE
Collect ORT issues and put them all next to tree root

### PR DESCRIPTION
**Pull request by sebathomas**

### CODE REVIEW

File: reporter/src/main/kotlin/reporters/OpossumReporter.kt [hunk 1]
```diff
@@ -44,15 +44,25 @@ import java.util.zip.Deflater
...
-        .replace(Regex("/*$"), if (isDir) {"/"} else {""})
-        .replace(Regex("/+"),"/")
+        .replace(
+            Regex("/*$"), if (isDir) {
+                "/"
+            } else {
+                ""
+            }
+        )
+        .replace(Regex("/+"), "/")
```
Comment (JakobSchubert) [inline reporter/src/main/kotlin/reporters/OpossumReporter.kt:60]: Just autoformatting. There should be no changes.

File: reporter/src/main/kotlin/reporters/OpossumReporter.kt [hunk 2]
```diff
@@ -317,10 +340,16 @@ class OpossumReporter : Reporter {
...
+            dependency: PackageReference,
+            curatedPackages: SortedSet<CuratedPackage>,
+            relRoot: String,
+            level: Int = 1
+        ) {
             val dependencyId = dependency.id
             log.debug("$relRoot - $dependencyId - Dependency")
-            val dependencyPath = pathResolve(listOf(relRoot, dependencyId.namespace, "${dependencyId.name}@${dependencyId.version}"))
+            val dependencyPath =
+                pathResolve(listOf(relRoot, dependencyId.namespace, "${dependencyId.name}@${dependencyId.version}"))
```
Comment (JakobSchubert) [inline reporter/src/main/kotlin/reporters/OpossumReporter.kt:352]: Just autoformatting

File: reporter/src/main/kotlin/reporters/OpossumReporter.kt [hunk 3]
```diff
@@ -349,13 +378,22 @@ class OpossumReporter : Reporter {
...
+            }
         }
 
-        fun addProject(project: Project, curatedPackages: SortedSet<CuratedPackage>, excludedScopes: Set<String>, relRoot: String = "/") {
+        fun addProject(
+            project: Project,
+            curatedPackages: SortedSet<CuratedPackage>,
+            excludedScopes: Set<String>,
+            relRoot: String = "/"
+        ) {
```
Comment (JakobSchubert) [inline reporter/src/main/kotlin/reporters/OpossumReporter.kt:396]: Just autoformatting.

**Comment by sebathomas** [comment]
Moved to https://github.com/opossum-tool/oss-review-toolkit/pull/7

**Pull request closed without merging**